lib/ex_doc/formatter/html.ex

```diff
@@ -384,10 +386,31 @@ defmodule ExDoc.Formatter.HTML do
     end)
   end
 
+  defp normalize_extras(base) when is_binary(base), do: {base, %{}}
+  defp normalize_extras({base, opts}), do: {base, Map.new(opts)}
```

Comment:
This is purely for pattern matching to flatten out `build_extra/5`. The alternative was a cond in `build_extra/5`, but this seemed a little clearer.